File: app/src/main/java/com/microlabs/trallet/view/AddBookActivityView.java
package com.microlabs.trallet.view;

import com.microlabs.trallet.model.Book;

/**
 * Created by Hansen on 5/12/2017.
 *
 * Add Book Activity View
 */

public interface AddBookActivityView {
    void showError();

    void done();

    void showBook(Book book);
}

File: app/src/main/java/com/microlabs/trallet/view/AddBookActivityView.kt
package com.microlabs.trallet.view

import com.microlabs.trallet.model.Book

/**
 * Created by Hansen on 5/12/2017.
 *
 * Add Book Activity View
 */

interface AddBookActivityView {
    fun showError()

    fun done()

    fun showBook(book: Book)
}
